# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1360162456 18000
#      Wed Feb 06 09:54:16 2013 -0500
# Node ID 7275cfaddb5e0f8c0a9c83aa5658880cb835b923
# Parent  3f93cd251bdbde161def8a0ebb4f646c2453a113
Fix texenhanced regression.

* scripts/private/__go_draw_axes__.m: Fix regression.

diff --git a/scripts/plot/private/__go_draw_axes__.m b/scripts/plot/private/__go_draw_axes__.m
--- a/scripts/plot/private/__go_draw_axes__.m
+++ b/scripts/plot/private/__go_draw_axes__.m
@@ -2345,17 +2345,17 @@ function str = no_super_sub_scripts (str
     str = char (labels);
   endif
 endfunction
 
 function str = __tex2enhanced__ (str, fnt, it, bld)
   persistent sym = __setup_sym_table__ ();
   persistent flds = fieldnames (sym);
 
-  [s, e, m] = regexp (str,'\\\\([a-zA-Z]+|0)','start','end','matches');
+  [s, e, m] = regexp (str, "\\\\([a-zA-Z]+|0)", "start", "end", "matches");
 
   for i = length (s) : -1 : 1
     ## special case for "\0"  and replace with "{/Symbol \306}'
     if (strncmp (m{i}, '\0', 2))
       str = cstrcat (str(1:s(i) - 1), '{/Symbol \306}', str(s(i) + 2:end));
     else
       f = m{i}(2:end);
       if (isfield (sym, f))
